Makefile

```diff
@@ -13,7 +13,9 @@ bootstrap: $(VENV)/bin/pip
 	$(VENV)/bin/pip install -e .[dev]
 
 format:
-	$(VENV)/bin/black .
+	$(VENV)/bin/codespell
+	$(VENV)/bin/ruff check
```

Comment:
I believe this needs to be `ruff check --fix` to perform changes, right?

Reply:
Some folks want ruff to autofix but others do not often because they are used to flake8 which never fixes.